packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -166,14 +166,13 @@ export const TranslationSidebar = ({
                   <RichTextInput
                     key={word.id}
                     name="noteContent"
-                    value={originalNoteContent}
+                    value={noteContent}
                     onBlur={async (e) => {
-                      setNoteContent(e.target.value);
-                      saveNote();
+                      saveNote(e.target.value);
+                      saveNote.flush();
                     }}
```

Comment:
On blur we can call `saveNote` again, but now we can also use `flush` to force it to execute immediately